update PR template (#4523)
PyTorchLightning/pytorch-lightning: .github/PULL_REQUEST_TEMPLATE.md

<!-- For CHANGELOG separate each item in unreleased section by a blank line to reduce collisions -->

## PR review
Anyone in the community is free to review the PR once the tests have passed.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In in short, see following bullet-list:  

 - [ ] Is this pull request ready for review? (if not, please submit in draft mode)
 - [ ] Check that all items from **Before submitting** are resolved

>  - [ ] Check that title is self explanatory and description well explains what is this PR about

Reviewer: ```suggestion
 - [ ] Make sure the title is self explanatory and the description concisely explains the PR
```
Just a small wording suggestion :]

>  - [ ] Add labels and milestones (and optionally projects) to the PR so it can be classified

Reviewer 2: Can you make this clearer? "Bugfixes should be including in bugfix release milestones (m.f.X), features should be included in (m.X.b) releases".
